app/lib/features/settings/widgets/settings_menu.dart

```diff
@@ -24,216 +23,227 @@ class SettingsMenu extends ConsumerWidget {
+              iconData: Atlas.trash_can_thin,
+              iconColor: Theme.of(context).colorScheme.error,
+              title: 'Deactivate Account',
+              subTitle: 'Irreversibly deactivate this account',
+              titleStyles: TextStyle(
+                color: Theme.of(context).colorScheme.error,
               ),
-              SettingsTile.navigation(
-                key: deactivateAccount,
-                title: Text(
-                  'Deactivate Account',
-                  style: TextStyle(
-                    color: AppTheme.brandColorScheme.error,
-                    //backgroundColor: AppTheme.brandColorScheme.onError,
-                  ),
-                ),
-                description: Text(
-                  'Irreversibly deactivate this account',
-                  style: TextStyle(
-                    color: AppTheme.brandColorScheme.error,
+              onTap: () => deactivationConfirmationDialog(context, ref),
+            ),
+          ],
+        ),
+        const SizedBox(height: 20),
+      ],
+    );
+  }
+
+  Widget _settingMenuSection({
+    required BuildContext context,
+    required String sectionTitle,
+    bool isDanderZone = false,
+    required List<Widget> children,
+  }) {
+    return Card(
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.circular(16),
+      ),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Padding(
+            padding: const EdgeInsets.only(left: 15.0, top: 10.0),
+            child: Text(
+              sectionTitle,
+              style: Theme.of(context).textTheme.labelLarge!.copyWith(
+                    color: isDanderZone
+                        ? Theme.of(context).colorScheme.error
+                        : null,
                   ),
-                ),
-                leading: Icon(
-                  Atlas.trash_can_thin,
-                  color: AppTheme.brandColorScheme.error,
-                ),
-                onPressed: (context) =>
-                    deactivationConfirmationDialog(context, ref),
-              ),
-            ],
+            ),
+          ),
+          Column(
+            children: children,
           ),
         ],
       ),
     );
   }
+
+  Widget _settingMenuItem({
+    Key? key,
+    required BuildContext context,
+    required IconData iconData,
+    Color? iconColor,
+    required String title,
+    TextStyle? titleStyles,
+    String? subTitle,
+    VoidCallback? onTap,
+    bool enable = true,
+  }) {
+    return Card(
+      child: ListTile(
+        key: key,
+        onTap: onTap,
+        leading: Icon(
+          iconData,
+          color: enable ? iconColor : Theme.of(context).disabledColor,
+        ),
+        title: Text(
+          title,
+          style: titleStyles?.copyWith(
+            color: enable ? null : Theme.of(context).disabledColor,
+          ),
+        ),
+        subtitle: subTitle == null
```

Comment:
Could be simplified to 
```suggestion
        subtitle: subTitle ?? Text(
                subTitle,
                style: titleStyles?.copyWith(
                  color: enable ? null : Theme.of(context).disabledColor,
                ),
              ),
```